spec/features/create_work_spec.rb

```diff
@@ -1,4 +1,4 @@
-RSpec.describe 'Creating a new Work', :js, :workflow do
+RSpec.describe 'Creating a new Work', :js, :workflow, :clean_repo do
```

Comment:
Did something in the PR cause the need for `:clean_repo`?

Reply:
If this fails once, then it continues to fail. The work gets created with the same title as previous attempts, so the page has multiple works with the same title, and a later find fails.